src/Microsoft.Health.Dicom.Core/Features/Indexing/ReindexStatus.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Indexing
+{
+    /// <summary>
+    /// The reindex status.
+    /// </summary>
+    public enum ReindexStatus
```

Comment:
Should this be indexing-specific? Or more general for operations?

Reply:
I'm good with indexing-specific -- although so far I'm not sure if we need `Paused` and `Completed` for deindexing. :)
My concern with promoting to operation level  is that 'Paused' is special, it's on `Tag` instead of `Operation`